src/dmd/dmangle.d

```diff
@@ -1027,15 +1027,15 @@ public:
         buf.writeByte(m);
         buf.print(q.length);
         buf.writeByte('_');    // nbytes <= 11
-        size_t qi = 0;
-        for (char* p = cast(char*)(*buf)[].ptr + buf.length, pend = p + 2 * q.length; p < pend; p += 2, ++qi)
+        const len = buf.length;
+        auto slice = buf.fillVoid(2 * q.length);
```

Comment:
This is now reserving twice (already in line 1026, which could be omitted).

Reply:
It's doing something fundamentally different from reserving. The space past `offset` should be off limits to users of `OutBuffer`. `fillVoid()` actually allocates that space to the user.

Reply:
I understand. I was referring to `fillVoid` reserving internally, while this is also done manually above. Doesn't do much harm, though.